python/grass/temporal/register.py

```diff
@@ -209,24 +245,18 @@ def register_maps_in_space_time_dataset(
             mapname = line_list[0].strip()
             row = {}
 
-            if start_time_in_file and end_time_in_file:
+            if start_time_in_file:
                 row["start"] = line_list[1].strip()
-                row["end"] = line_list[2].strip()
 
-            if start_time_in_file and not end_time_in_file:
-                row["start"] = line_list[1].strip()
+            if end_time_in_file:
+                row["end"] = line_list[2].strip()
```

Comment:
I lost a track why this change is here.

Reply:
I just stubled of those lines of could that could be simplified as I did. No logical change, just one line of code and two conditions less to check...